BrCreatePersonaWizard.vue

```diff
@@ -76,7 +76,7 @@ export default {
             size: '65px',
             color: 'primary'
           },
-          heading: 'Let\'s name your Persona',
+          heading: 'Let\'s name your Persona and choose a color for it',
```

Comment:
This heading is wordy. Maybe, "Let's personalize your profile." We should also be aware we are deprecating "Persona" for "Profile".